docs/workloads/azure/data/etl_pipelines/etl_intro_data_azure.md

```diff
@@ -15,29 +15,28 @@ keywords:
   - medallion
 ---
 
-Stacks Data Platform contains a range of example data workloads and pipelines, designed to demonstrate its capabilities
-for delivering production ready data engineering and analytics solutions. These example workloads cover all stages from
+The Stacks Data Platform solution accelerates the generation of production-ready ETL / ELT pipelines and workloads. New data engineering workloads can be generated through the [Datastacks] utility, while a range of example data workloads and pipelines are also provided. These workloads cover all stages from
 ingesting data from source, applying data transformations across data lake layers, ultimately enabling end-user data
 visualisations and analytics.
 
-## Data pipelines
+## Medallion architecture
 
 The data lake structure and approach used in Stacks is based upon the
 [medallion architecture](https://www.databricks.com/glossary/medallion-architecture) design pattern. This approach
 delivers a lakehouse solution, with the following structured data transformation layers:
 
-* [Bronze (data ingestion)](ingest_data_azure.md): Landing of raw data from source.
-* [Silver](silver_data_azure.md): Conformance, cleansing and data quality processing stage.
+* [Bronze](ingest_data_azure.md): Landing of raw data from source.
+* [Silver](data_processing.md): Conformance, cleansing and data quality processing stage.
 * Gold: Data aggregations, refinement and enrichment, producing datasets optimised for analytics and visualisation.
```

Comment:
Worth highlighting this is not available yet?
```suggestion
* Gold (Coming soon): Data aggregations, refinement and enrichment, producing datasets optimised for analytics and visualisation.
```

Reply:
The idea is that the data processing pipelines can be for either bronze -> silver, or silver -> gold, hopefully these updates clarify this. Once the templating is in place, you'll be able to generate either. This section is just describing the purpose of the layers in a medallion structure so think it's OK.
The element we don't yet have is the query layer over the top of gold (i.e. Fabric / Synpase). Don't think the docs refer to this part yet.